tests/components/google_assistant/test_trait.py

```diff
@@ -101,6 +103,20 @@
 )
 
 
+@pytest.fixture(autouse=True)
+async def _load_translations(hass: HomeAssistant) -> None:
+    """Override loading translations once."""
+
+    for platform in Platform:
+        hass.config.top_level_components.add(platform)
+    hass.config.top_level_components.add(input_select.DOMAIN)
+    hass.config.top_level_components.add(const.DOMAIN)
+    for language in BASIC_CONFIG.languages:
```

Comment:
How do i load non english language during tests? We don't expect any such language to exist i think. Maybe i could have a missing language and add that during tests?